app-crypt/stegdetect/stegdetect-0.6.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A Steganography Program"
+DESCRIPTION="Stegdetect is an automated tool for \
+detecting steganographic content in images."
+HOMEPAGE="https://github.com/abeluck/stegdetect"
+COPYRIGHT="2002 Niels Provos"
+LICENSE="BSD (4-clause)"
+REVISION="1"
+srcGitRev="28a4f074a71c682581314491224f9f41511c82e7"
+SOURCE_URI="https://github.com/abeluck/stegdetect/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="eb39bfcf50116c721c3bf0efaa168cb84b5141563b87d66235f1d21f3336e695"
+SOURCE_FILENAME="stegdetect-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="stegdetect-$srcGitRev"
+#PATCHES="stegdetect-$portVersion.patch"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	stegdetect = $portVersion
+	cmd:stegdetect = $portVersion
+	"
+REQUIRES=" 
+	haiku >= $haikuVersion
+	lib:libjpeg
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libjpeg
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc
+	cmd:make
+	cmd:awk
+	"
+
+BUILD()
+{
+	runConfigure --omit-dirs "docDir manDir" ./configure 
```

Comment:
```diff
 BUILD()
 {
+	autoreconf -fi
- 	runConfigure --omit-dirs "docDir manDir" ./configure
+	ac_cv_func_dirname=no \
+	./configure --prefix="$prefix" \
+		--includedir="$includeDir" \
+		--mandir="$manDir"
 	make $jobArgs
}
```
We could use runConfigure but we would have to pass more parameters to `--omit-dirs` like dataDir, dataRootDir and probably several others.

BTW, I added a line with `ac_cv_func_dirname=no \` to tell configure to skip one of the tests because it was crashing.
If you find out that configure does not crash for you then we can remove that line.